code-studio/src/js/initApp/project.js

```diff
@@ -279,10 +279,12 @@ var projects = module.exports = {
   /**
    *
    * @param {Object} sourceHandler Object containing callbacks provided by caller.
-   * @param {Function} sourceHandler.setInitialLevelHtml
-   * @param {Function} sourceHandler.getLevelHtml
-   * @param {Function} sourceHandler.setInitialLevelSource
-   * @param {Function} sourceHandler.getLevelSource
+   * @param {function(string)} sourceHandler.setInitialLevelHtml
+   * @param {function(): string} sourceHandler.getLevelHtml
+   * @param {function(string)} sourceHandler.setInitialLevelSource
+   * @param {function(): string} sourceHandler.getLevelSource
+   * @param {function(SerializedAnimationList)} sourceHandler.setInitialAnimationList
+   * @param {function(function(): SerializedAnimationList)} sourceHandler.getAnimationList
```

Comment:
Redocumented according to [these closure compiler guidelines](https://github.com/code-dot-org/code-dot-org/blob/6a937299389fe541185b39a13b09e1beba6f4347/apps/src/gamelab/PropTypes.js) for documenting function types.  It seemed important now that `getAnimationList` is significantly different from the other two getters.